infrastructure/hub-and-spoke-peerings/README.md

```diff
@@ -0,0 +1,71 @@
+# Hub and Spoke via VPC Peering
+
+This sample creates a simple **Hub and Spoke** architecture, where the VPC network connects satellite locations (spokes) through a single intermediary location (hub) via [VPC Network Peering](https://cloud.google.com/vpc/docs/vpc-peering). 
```

Comment:
Since you added a section below on benefits, I would move the NOTE paragraph below the benefits, and list drawbacks. The drawbacks also *need* to mention the complications with managed services (GKE, Cloud SQL, etc.) that use peering. Let me know if you want me to work on the README.